src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Roller.java

```diff
@@ -101,27 +82,33 @@ public boolean hasGamePiece() {
     }
 
     public double getGamePieceDistanceIn() {
-        return Units.metersToInches((distSensor.getRange() - distSensorDepthMM /* The sensor measures from the back of the sensor */) /1000 /* Convert mm to m */);
+        return Units.metersToInches((distSensor.getRange() - distSensorDepthMM /*
+                                                                                * The sensor measures from the back of
+                                                                                * the sensor
+                                                                                */) / 1000 /* Convert mm to m */);
```

Comment:
Please undo this formatting change or adjust it so that individual comments do not span lines and there's not 18 tabs